test/metabase/api/card_test.clj

```diff
@@ -2274,7 +2274,7 @@
           (when before-fn
             (before-fn (db/select-one Card :id card-id)))
           (let [result (mt/user-http-request user :put status-code (str "card/" card-id) {:is_write new-value})]
-            (result-fn result))
+            (result-fn result new-value))
```

Comment:
Below there seem to be some calls passing a single arity function as `result-fn`.